<robot name="body_reference">
  <link name="body_reference_link">
    <inertial>
      <origin xyz="0 0 0" />
      <mass value="1e-5" />
      <inertia  ixx="1e-5" ixy="0.0"  ixz="0.0"  iyy="1e-5"  iyz="0.0"  izz="1e-5" />
    </inertial>
    <visual>
      <origin xyz="0 0 0"/>
      <geometry>
        <box size="0.8 0.3 0.1"/>
    </geometry>
    </visual>
  </link>
  <gazebo reference="body_reference_link">
    <material>Gazebo/RedTransparent</material>
  <static>true</static>
    <turnGravityOff>true</turnGravityOff>      
  </gazebo>
</robot>